index.html

```diff
@@ -158,13 +189,14 @@ <h1 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h1>
                 <hr class="line">
             </li>
         </ul>
+     </div>
     </section>
     <!--contact-us-form-->
     <section class="contact-us">
```

Comment:
Good job here as well your form is centered and section looks good on desktop , **but** Kindly check the background image in this section on figma design and make sure to add it and replicate the given template as much as possible

![Uploading figma contact form background image.PNG…]()